lte/gateway/c/core/oai/tasks/nas/esm/DedicatedEpsBearerContextActivation.c

```diff
@@ -427,8 +427,8 @@ status_code_e dedicated_eps_bearer_activate_t3485_handler(
   ue_mm_context_t* ue_mm_context = mme_app_get_ue_context_for_timer(
       ue_id, "EPS BEARER DEACTIVATE T3495 Timer");
   if (ue_mm_context == NULL) {
-    OAILOG_ERROR(
-        LOG_MME_APP,
+    OAILOG_ERROR_UE(
```

Comment:
this won't work since `ue_mm_context` is null, let's keep it as before

Reply:
Resolved